django/tests/chat/test_chat_views.py

```diff
@@ -218,16 +218,18 @@ async def stream_generator():
     )
     # Iterate over the response_stream generator
     final_output = ""
-    first = True
+    message_counter = 0
```

Comment:
Previously, hitting the Stop button would cause htmx_stream to terminate AFTER appending one last response. Now, responses only get appended if the stop button hasn't been pressed yet. Thus, I had to change this test to send the Stop message AFTER the "second thing"